src/main/java/kg/jarkyn/cobspecserver/Server.java

```diff
@@ -0,0 +1,29 @@
+package kg.jarkyn.cobspecserver;
+
+public class Server {
+    private Listener listener;
+    private ResponseController responseController;
+
+    public Server(Listener listener, ResponseController responseController) {
+        this.listener = listener;
+        this.responseController = responseController;
+    }
+
+    public void run() {
+        while (listener.isAccepting()) {
+            satisfyRequest();
+        }
+    }
+
+    public void satisfyRequest() {
+        respond(listen());
```

Comment:
I like the short methods in this class. Here you could go even further with something like `respondTo(nextClient())` to make it more readable.